lib/chef/chef_fs/chef_fs_data_store.rb

```diff
@@ -468,6 +500,12 @@ def list(path)
       def exists?(path)
         if use_memory_store?(path)
           @memory_store.exists?(path)
+
+        # /policy_groups/NAME/policies/POLICYNAME
+        elsif path[0] == "policy_groups" && path[2] == "policies" && path.length == 4
```

Comment:
Not worth doing now, but this kind of path-checking makes me wish again for a more comprehensible function that matches the `path` array against a string like `/policy_groups/*/policies/*`.